guide/how-it-works/lightning-service-providers.md

```diff
@@ -0,0 +1,310 @@
+   height = 740
+   layout = "full-width"
+%}
+
+# Lightning Service Providers
+
+The [lightning network]({{ '/guide/getting-started/technology-primer/#the-lightning-payment-network/' | relative_url }}) is currently bitcoins best solution at [scaling]({{ '/guide/getting-started/technology-primer/#do-all-transactions-have-to-be-this-secure/' | relative_url }}) to billions of people. However, using the lightning network self-custodially brings about many new usability issues, such as channel management, routing, uptime requirements and backups.
+
+Lightning service providers (LSPs) solve these user friction points by providing services that make it easy to access and use the lightning network. They act as trust-minimized third parties that make it easy to onboard new user into lightning.
+
+Roy Sheinfeld from [Breez](https://breez.technology/) explains LSPs in more depth in his [introducing lightning service providers](https://medium.com/breez-technology/introducing-lightning-service-providers-fe9fb1665d5f) and [envisioning LSPs in the lightning economy](https://medium.com/breez-technology/envisioning-lsps-in-the-lightning-economy-832b45871992) articles.
+
+## What is a lightning service?
+
+Lightning services are trust-minimized offerings by LSPs that solve user pain points when using a non-custodial bitcoin lightning wallet.
+
+A lightning service should always non-custodial, be opt-out, only used when necessary and any trade-offs should be clearly communicated to users.
+
+Below are some lightning services, the pain points they solve, their trade-offs, and the applications that use them.
+
+## Backup & recovery
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/how-it-works/lightning-service-providers/backups-recovery.jpg"
+   retina = "/assets/images/guide/how-it-works/lightning-service-providers/backups-recovery@2x.jpg"
+   alt-text = "Backups & recovery LSP service image."
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+A Lightning wallet requires both a recovery phrase and up-to-date payment channel states to be recovered. Payment channel states need to be backed up each time a payment is received or sent. Users can make these backups, though if done incorrectly or stored insecurely, users are at risk of losing their bitcoin.
+
+A backup & recovery service automatically backs up and stores a users channel states with the LSP when a payment is sent or received. These backups are encrypted, usually by the user’s recovery phrase, before being sent to the LSP.
+
+When a user recovers their wallet, they enter their recovery phrase, the LSP sends them the encrypted channel states, these are decrypted by the user, and the wallet is restored.
+
+This service requires trust that the LSP will make accurate, and securely store, the users channel state backups. Trusting your backups with a single third-party like this is also a security risk and makes users reliant on the LSP.
```

Comment:
This sentence felt like it talked to the user instead of app maker, little update  to make sure we're speaking to the target audience of ours.
```suggestion
This service requires trust that the LSP accurately and securely stores the user's channel state backups. Entrusting a backup with a single third-party like this is also a security risk and makes users reliant on the LSP.
```